apps/artifactory/artifactory-ha/install.yaml

```diff
@@ -78,7 +78,7 @@
       # make secret
     - name: Make the Patroni secrets (processing)
       command: >
-        oc process -f ./templates/patroni-prereq.yaml \
+        oc process -f ./templates/patroni-prereq-1.yaml \
```

Comment:
but this will still break the second time you run? maybe turn into a parameter?

Reply:
why would this still break? it's just processing, not applying :)